src/services/config/config.service.ts

```diff
@@ -322,6 +333,10 @@ export class ConfigService implements ConfigInterface {
         }
     }
 
+    private async getGAToken(): Promise<string> {
+        return (await this.tokenHttpService.getGAToken()).token;
```

Comment:
Maybe
```suggestion
        return (await this.tokenHttpService.getGAToken())?.token;
```
would be more safe

Reply:
https://github.com/bastionzero/zli/pull/304/commits/d018c950cff0dacd67575e85a499fd5c710d8650